docs/metrics/getting-started-histogram/README.md

```diff
@@ -34,16 +34,16 @@ output from the console, similar to shown below:
 ```text
 Export 14:30:58.201 14:30:59.177 histogram [tag1=value1;tag2=value2] Histogram, Meter: TestMeter/0.0.1
 Value: Sum: 33862 Count: 62
-(-? - 0) : 0
-(0 - 5) : 0
-(5 - 10) : 0
-(10 - 25) : 2
-(25 - 50) : 0
-(50 - 75) : 1
-(75 - 100) : 1
-(100 - 250) : 6
-(250 - 500) : 18
-(500 - 1000) : 34
+(-? - 0] : 0
```

Comment:
I think it is something like this (in the latest main branch):

```text
(-Infinity, 0]: 0
(0, 5]: 2
(5, 10]: 1
(10, 25]: 5
(25, 50]: 7
(50, 75]: 5
(75, 100]: 8
(100, 250]: 40
(250, 500]: 90
(500, 1000]: 155
(1000, +Infinity): 0
```

Reply:
Yes, you are right. I was unsure whether the -Infinity/+Infinity representation was due to my local encoding or not. Let me push a new iteration. Thanks!

Reply:
I think [this PR](#2285) changed it to use ASCII output of IEEE754 `+Infinity` and `-Infinity`, so we've avoided the code page issue. https://github.com/open-telemetry/opentelemetry-dotnet/blob/e27070d2204b82a1f19c43524ab4ed4394d3a3f3/src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs#L96